### PR TITLE
feat(grpc): add fee config to get node info api response

### PR DESCRIPTION
**Pull request by alidevjimmy**
## Description

This PR adds a node fee config to `GetNodeInfoResponse` of `GetNodeInfo` method.

## Related issue(s)

- #1546

### CODE REVIEW

File: node/node.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func NewNode(genDoc *genesis.Genesis, conf *config.Config,
 	if conf.GRPC.BasicAuth != "" {
 		enableHTTPAuth = true
 	}
-	grpcServer := grpc.NewServer(conf.GRPC, st, syn, net, consMgr, walletMgr)
+	grpcServer := grpc.NewServer(conf.GRPC, st, syn, net, consMgr, walletMgr, conf.TxPool)
```
Comment (Ja7ad) [inline node/node.go:94]: @b00f What you think we pass just `conf` instead part of configuration?

**Comment by Ja7ad** [comment]
@alidevjimmy In http server you need add fee configuration to node handler.

https://github.com/pactus-project/pactus/blob/80ea4db1644d6e552bcf216541db9a1b63512f6a/www/http/network.go#L80-L118

File: www/grpc/proto/network.proto [hunk 2]
```diff
@@ -129,3 +131,13 @@ message CounterInfo {
   // Total number of bundles.
   uint64 Bundles = 2;
 }
+
+// FeeConfig holds the fee configuration that stored in node config.
+message FeeConfig {
+  // Fixed fee for each transaction.
+  double fixed_fee = 1;
+  // Number of bytes an account can send each day without paying a fee.
```
Comment (b00f) [inline www/grpc/proto/network.proto:139]: Please add this comment here: `The `daily_limit` is part of he consumptional fee model.`

File: www/grpc/proto/network.proto [hunk 3]
```diff
@@ -129,3 +131,13 @@ message CounterInfo {
...
+  uint32 daily_limit = 2;
+  // Fee per byte in PAC.
```
Comment (b00f) [inline www/grpc/proto/network.proto:141]: Please add this comment here: `The `unit_price` is part of he consumptional fee model.`

**Review by b00f — CHANGES_REQUESTED** [review]

File: node/node.go (hunk 1 above)
Comment (b00f) [reply node/node.go:94]: Nope! We can't add parameter that simple!
Try to find another way...

Comment (b00f) [reply node/node.go:94]: Let's think out of box; gRPC almost have everything. Right?
Why don't we simply give him Node.

Comment (Ja7ad) [reply node/node.go:94]: We can give access to configuration via node?

Comment (alidevjimmy) [reply node/node.go:94]: node itself depends on gRPC

Comment (b00f) [reply node/node.go:94]: Node has all modules...

**Comment by alidevjimmy** [comment]
> @alidevjimmy In http server you need add fee configuration to node handler.
> 
> https://github.com/pactus-project/pactus/blob/80ea4db1644d6e552bcf216541db9a1b63512f6a/www/http/network.go#L80-L118

It's Done.

File: node/node.go (hunk 1 above)
Comment (alidevjimmy) [reply node/node.go:94]: If we pass node to gRPC we will encounter **cycle import**.

**Comment by b00f** [comment]
Thanks @alidevjimmy  but This PR only adds complexity to the code, without any clear reason. Let's close it.
In case we really need this information, we can think about it.

**Pull request closed without merging**